next.config.js

```diff
@@ -2,4 +2,8 @@ module.exports = {
   images: {
     domains: [process.env.NEXT_IMAGE_DOMAIN],
   },
+  i18n: {
```

Comment:
adding here the same setup as we have in drupal, where we have two locales, and the "en" one is the default.